src/bosh_azure_cpi/lib/cloud/azure/azure_client.rb

```diff
@@ -2042,7 +2042,7 @@ def http(uri)
 
     # https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-protocols-oauth-service-to-service
     def get_token(force_refresh = false)
-      if @token.nil? || (Time.at(@token['expires_on'].to_i) - Time.now) <= 0 || force_refresh
+      if @token.nil? || (Time.at(@token['expires_on'].to_i) - Time.new) <= 0 || force_refresh
```

Comment:
It seems that `Time.now` is same as `Time.new` without arguments. Why we change it to `Time.new`?

Reply:
because do not want to mock both now and new. 
so change them to all new.

Reply:
There are only `Time.now` in the original codes. No need to mock `Time.new`. no?

Reply:
Actually, there are `Time.new` in the original codes (`blob_manager.rb`). It makes sense to use change `Time.now` to `Time.new`, because we don't want to mock bosh `now` and `new`.